docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  web:
+    build: .
+    command: python /code/forum/manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/code
+    ports:
+      - 8000:8000
+    depends_on:
+      - db
```

Comment:
Define environment variables like `DJANGO_SECRET_KEY`, `DJANGO_DEBUG`, etc., either in the `docker-compose.yml` file (for development only) or in a `.env` file for better separation of concerns.